### PR TITLE
code-copy-button

### PR DESCRIPTION
**Pull request by ntorbinskiy**

### CODE REVIEW

**Comment by alexander-vashchuk** [comment]
@ntorbinskiy 
???

**Comment by 1alexvash** [comment]
@ntorbinskiy 

???

File: styles/main.scss [hunk 1]
```diff
@@ -1405,4 +1405,49 @@ blockquote p:last-child {
 .remark-highlight {
   overflow: auto;
   max-width: calc(100vw - 30px);
+  position: relative;
+}
+
+.btn_copy {
```
Comment (alexander-vashchuk) [inline styles/main.scss:1411]: @ntorbinskiy do we name classes like this in the project?

Comment (ntorbinskiy) [reply styles/main.scss:1411]: @1alexvash @alexander-vashchuk 
You mean, that I use <<_>>, not <<->> in the names of classes?

**Pull request closed without merging**